executor/src/transactions.rs

```diff
@@ -277,19 +328,95 @@ impl TransactionBuilder {
         if length_estimate > PACKET_DATA_SIZE {
             self.start_from_alt(tx_data, None).await
         } else {
-            self.execute_from_data_emulated(tx_data, chain_id, None)
+            self.dispatch_data_execution_by_version(tx_data, chain_id, None)
                 .await
         }
     }
 
-    async fn execute_from_data_emulated(
+    async fn dispatch_data_execution_by_version(
         &self,
         tx_data: TxData,
         chain_id: u64,
         alt: Option<AltInfo>,
     ) -> anyhow::Result<OngoingTransaction> {
-        let fallback_iterative =
-            |tx_data, alt| async move { self.empty_holder_for_iterative_data(tx_data, alt).await };
+        // NOTE: Use the last version with the old behaviour to support pre release version.
+        // (e.g. 1.0.0-dev < 1.0.0)
+        const BREAKPOINT: Version = Version::new(1, 14, 5);
+
+        if self.version > BREAKPOINT {
```

Comment:
but what if they release 1.14.6 🤔  ?

Reply:
how about `1.14.u64::MAX`?

Reply:
why not >= 1.15 ?

Reply:
`1.15.0-dev < 1.15.0` see [this](https://docs.rs/semver/latest/semver/struct.Version.html#total-ordering)